### DISCUSSION TITLE
Did Podman compatibility with Debian Buster (10) just disappear?

### DISCUSSION THREAD

**Discussion by kjohnson0451** [containers/podman#12109] [Q&A] [answered]
As of even yesterday (10/25/2021), I was able to install Podman on Debian Buster like so
```
# Install and configure podman
RUN echo 'deb http://deb.debian.org/debian buster-backports main' >> /etc/apt/sources.list \
    && echo 'deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/Debian_10/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list \
    && curl -L https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/Debian_10/Release.key | apt-key add - \
    && apt-get update -qq \
    && apt-get install -qq -y uidmap \
    && apt-get -t buster-backports install -qq -y libseccomp2 \
    && apt-get -qq -y install podman \
    && apt-get install -y iptables
COPY containers.conf /usr/share/containers/containers.conf
```

I realize the Debian section in the [installation instructions](https://podman.io/getting-started/installation) has changed to a simple `sudo apt-get -y install podman`, but that only applies to Debian Bullseye (11). Is there no way to install Podman on Debian Buster anymore?

**Comment by mheon** [comment]
@lsm5 PTAL

**Comment by kjohnson0451** [comment, marked as answer]
The installation commands in the OP work again. I know they're not the official instructions anymore, and I should move onto Debian Bullseye, but I'm glad it's not broken for Debian 10 for now nonetheless.
